NDTensors/ext/NDTensorscuTENSORExt/contract.jl

```diff
@@ -0,0 +1,22 @@
+using NDTensors: NDTensors, DenseTensor, array
+using NDTensors.Expose: Exposed, unexpose
+using cuTENSOR: CuArray, CuTensor, mul!
+
+function NDTensors.contract!(
+  R::Exposed{<:CuArray,<:DenseTensor},
+  labelsR,
+  T1::Exposed{<:CuArray,<:DenseTensor},
+  labelsT1,
+  T2::Exposed{<:CuArray,<:DenseTensor},
+  labelsT2,
+  α::Number=one(Bool),
+  β::Number=zero(Bool),
+)
+  cuR, cuT1, cuT2 =
+    CuTensor.(
+      array.((unexpose(R), unexpose(T1), unexpose(T2))),
+      collect.((labelsR, labelsT1, labelsT2)),
+    )
```

Comment:
I think it would be better to split this into three separate lines, I think it will be easier to read.

Reply:
That makes sense! I split these into three lines, thanks!